### PR TITLE
Usage Docs link added to README and Docs Front page

### PR DESCRIPTION
**Pull request by canyon289**
Will merge after checks are done. I can't imagine these changes will be very controversial and I'd like to send Stefan an email by tonight.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -13,6 +13,14 @@ Includes functions for posterior analysis, model checking, comparison and diagno
 The official ArviZ documentation can be found here
 https://arviz-devs.github.io/arviz/index.html
```
Comment (ColCarroll) [inline README.md]: Not your change, not entirely necessary, but can we change this to an inline link, and combine these three short sentences into one paragraph (they'll need periods at the end of each)? Here's the link I mean:

The official ArviZ documentation can be found [here](https://arviz-devs.github.io/arviz/index.html).

File: README.md [hunk 2]
```diff
@@ -10,8 +10,10 @@ Includes functions for posterior analysis, model checking, comparison and diagno
 
 ## Documentation
 
-The official ArviZ documentation can be found here
-https://arviz-devs.github.io/arviz/index.html
+The ArviZ documentation can be found in the [official docs](https://arviz-devs.github.io/arviz/index.html) First time users may find the [quickstart](https://arviz-devs.github.io/arviz/notebooks/Introduction.html).
```
Comment (ColCarroll) [inline README.md]:
```suggestion
The ArviZ documentation can be found in the [official docs](https://arviz-devs.github.io/arviz/index.html). First time users may find the [quickstart](https://arviz-devs.github.io/arviz/notebooks/Introduction.html)
```

File: README.md [hunk 3]
```diff
@@ -10,8 +10,10 @@ Includes functions for posterior analysis, model checking, comparison and diagno
...
+to be helpful. Additional guidance can be found in the
+[usage documentation](https://arviz-devs.github.io/arviz/usage.html)
```
Comment (ColCarroll) [inline README.md]:
```suggestion
[usage documentation](https://arviz-devs.github.io/arviz/usage.html).
```

**Comment by canyon289** [comment]
Incorporated feedback. I also changed the link text to be more descriptive than `here` because the browser turns the link text blue and makes it stand out.
arvizreadme

File: README.md (hunk 3 above)
Comment (canyon289) [reply README.md]: Ugh getting sloppy. Thank you. After this I'm done for tonight :)

Comment (ColCarroll) [reply README.md]: :+1: i can merge in a few minutes -- thanks much!

**Pull request merged**